aesthetics/fisher/fisher.py

```diff
@@ -7,18 +7,15 @@
 from collections import OrderedDict
 from concurrent.futures import ProcessPoolExecutor
 from scipy.stats import multivariate_normal
-
+descriptors = None
```

Comment:
Lint: There should be one or more lines b/w imports and code

Reply:
Alright 👍